NeoML/Python/neoml/Blob.py

```diff
@@ -20,102 +20,137 @@
         """
         return self._internal.math_engine()
 
     @property
     def shape(self):
-        """
+        """The blob shape: the array of 7 int elements with dimension lengths.
         """
         return self._internal.shape()
 
     @property
     def batch_len(self):
-        """
+        """Returns the BatchLength dimension.
         """
         return self._internal.batch_len()
 
     @property
     def batch_width(self):
-        """
+        """Returns the BatchWidth dimension.
         """
         return self._internal.batch_width()
 
     @property
     def list_size(self):
-        """
+        """Returns the ListSize dimension.
         """
         return self._internal.list_size()
 
     @property
     def height(self):
-        """
+        """Returns the Height dimension.
         """
         return self._internal.height()
 
     @property
     def width(self):
-        """
+        """Returns the Width dimension.
         """
         return self._internal.width()
 
     @property
     def depth(self):
-        """
+        """Returns the Depth dimension.
         """
         return self._internal.depth()
 
     @property
     def channels(self):
-        """
+        """Returns the Channels dimension.
         """
         return self._internal.channels()
 
     @property
     def size(self):
-        """
+        """Returns the blob total size: the product of all dimensions.
         """
         return self._internal.size()
 
     @property
     def object_count(self):
-        """
+        """Returns the number of objects in the blob, 
+        equal to BatchLength * BatchWidth * ListSize.
         """
         return self._internal.object_count()
 
     @property
     def object_size(self):
-        """
+        """Returns the size of one object, equal to
+        Height * Width * Depth * Channels.
         """
         return self._internal.object_size()
 
     def asarray(self, copy=False):
-        """
+        """Returns the contents of the blob as a multi-dimensional array, 
+        keeping only those dimensions that are more than 1 element long.
+        
+        Parameters
+        ---------
+        copy : bool, default=False
+            If True, the data will be copied. If False, the array will share
+            the memory buffer with the blob and only provide more convenient 
+            access to the same data.
         """
         if type(self.math_engine) is MathEngine.GpuMathEngine:
             copy = True
         return numpy.array(self._internal, copy=copy)
 
     def copy(self, math_engine):
-        """
+        """Creates a blob copy independent of this blob.
         """
         return Blob(self._internal.copy(math_engine._internal))
 
 # -------------------------------------------------------------------------------------------------------------
 
 def store(blob, file_path):
-    """
+    """Stores the contents of the blob into a file at the specified path.
```

Comment:
_(Arguable)_ It stores the whole blob (not only contents, but also a shape and other info, which will allow to load it below)